packages/hadron-build/lib/generate-package-lock.js

```diff
@@ -86,26 +97,14 @@ async function main() {
     }
   );
 
-  await withProgress(
-    `Writing package lock file to ${path.relative(
-      process.cwd(),
-      path.join(workspacePath, 'package-lock.json')
-    )}`,
-    async () => {
-      // https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#file-format
```

Comment:
I think it helps to have a reference to the format so close, maybe we can keep it somehow?

Reply:
ah i don't think i removed this on purpose, will get it back!